test/specs/mainnet/01_TransferringFunds/Arbitrum.spec.js

```diff
@@ -2,25 +2,25 @@ import { PrimeSdk } from "@etherspot/prime-sdk";
 import { ethers } from "ethers";
 import { assert } from "chai";
 import { ERC20_ABI } from "@etherspot/prime-sdk/dist/sdk/helpers/abi/ERC20_ABI.js";
+import abi from "../../../data/NFTabi.json" assert { type: "json" };
 import * as dotenv from "dotenv";
 dotenv.config(); // init dotenv
 
 let arbitrumMainNetSdk;
 let arbitrumEtherspotWalletAddress;
 let sender = "0x97EC9803387fcBc4A238BCeA2C6C1b75438D39C3";
+let incorrectSender = "0x97ECC803387fcBc44238BCeA2CCC1b75433D39C0";
```

Comment:
Can we please maintain a common constants file across all scripts and add these values there? That will be easier to maintain.

Reply:
Created a file for the constants and modified all the test cases.